src/Miki/Miki.csproj

```diff
@@ -45,7 +45,7 @@
     <PackageReference Include="Miki.Framework.Arguments" Version="2.3.0" />
     <PackageReference Include="Miki.Framework.Commands" Version="2.3.1" />
     <PackageReference Include="Miki.Framework.Commands.Filters" Version="2.3.0" />
-    <PackageReference Include="Miki.Functional" Version="0.1.1" />
+    <PackageReference Include="Miki.Functional" Version="0.1.3" />
```

Comment:
Added support for results and Result.From(Func<>)

Reply:
Nice addition